Sources/UIView+PinLayout.swift

```diff
@@ -22,21 +22,11 @@ import Foundation
 #if os(iOS) || os(tvOS)
 import UIKit
 
-public extension UIView {
-    public var pin: PinLayout {
-        return PinLayoutImpl(view: self, keepTransform: true)
-    }
-
-    public var pinFrame: PinLayout {
-        return PinLayoutImpl(view: self, keepTransform: false)
-    }
-
-    @objc public var anchor: AnchorList {
-        return AnchorListImpl(view: self)
-    }
-
-    @objc public var edge: EdgeList {
-        return EdgeListImpl(view: self)
+extension UIView: Layoutable, SizeCalculable {
+    public typealias View = UIView
+    
+    public func convert(_ point: CGPoint, toView view: UIView) -> CGPoint {
```

Comment:
I'm not sure to understand why does this method is required? Why not simply use directly the method `convert(:CGPoint, to: UIView?)` 🤔

Reply:
I initially thought it was a collision with the function taking `UISpaceCoordinates` as a parameter but after investigation, it was because the view parameter is optional. Fixed.